Resources/input/fsh/ISiKDokumentenMetadaten.fsh

```diff
@@ -2,6 +2,22 @@ Profile: ISiKDokumentenMetadaten
 Parent: DocumentReference
 Id: ISiKDokumentenMetadaten
 Title: "Erforderliche Metadaten für Dokumentenaustausch in ISiK"
+Description: 
+  "Die nachfolgenden Festlegungen spezifizieren die Minimalanforderungen für die Bereitstellung von Dokumentenmetadaten im Rahmen des Bestätigungsverfahrens der gematik.  
```

Comment:
question: das entspricht dem bisherigen Test aus dem IG, d.h. wäre die Konsequenz nicht dies aus dem IG zu entfernen, um Redundanz zu vermeiden? Ich sehe allerdings keine Änderung am .md ...

issue: Gleichzeitig klingt der erste Satz nicht passend für eine Beschreibung des Profils.

suggestion: "Dieses Profil spezifiziert die Minimalanforderungen... "

Reply:
Ich würde die komplette Überarbeitung der Darstellung im IG (und Entfernung sämtlicher Redundanzen) dann in dem neuen (kopierten) IG angehen und dort dann sämtliche Doku per FQL aus den Ressourcen generieren lassen. Ich will aber beide IGs parallel laufen lassen, damit wir am Ende nochmal abgleichen können, dass nichts verloren gegangen ist.

Reply:
Formulierung angepasst